config/jobs/image-pushing/k8s-staging-e2e-test-images.sh

```diff
@@ -56,6 +56,16 @@ EOF
 for image in "${IMAGES[@]}"; do
     cat >>"${OUTPUT}" <<EOF
     - name: post-kubernetes-push-e2e-${image//\//-}-test-images
+      rerun_auth_configs:
```

Comment:
This should be singular. See this GoDoc: https://godoc.org/k8s.io/test-infra/prow/config#JobBase